keyboards/tn_alice/keymaps/default/keymap.c

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 QMK / MudkipMao
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[0] = LAYOUT(
+		KC_ESC,  KC_GRV,  KC_1, KC_2, KC_3,   KC_4,   KC_5, KC_6, KC_7, KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, 
+		KC_PGUP, KC_TAB,  KC_Q, KC_W, KC_E,   KC_R,   KC_T, KC_Y, KC_U, KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, 
+		KC_PGDN, KC_CAPS, KC_A, KC_S, KC_D,   KC_F,   KC_G, KC_H, KC_J, KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT, 
+		         KC_LSFT, KC_Z, KC_X, KC_C,   KC_V,   KC_B, KC_B, KC_N, KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_UP,   MO(2), 
+		         KC_LCTL, KC_LGUI,    KC_LALT,  KC_SPC,  KC_SPC,    MO(1),                        KC_LEFT, KC_DOWN, KC_RGHT
+	),
+
+	[1] = LAYOUT(
+		KC_TRNS, KC_TRNS, KC_F1,   KC_F2,   KC_F3,    KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,    KC_F10,  KC_F11,  KC_F12,  QK_BOOT, 
+		KC_TRNS, KC_TRNS, RGB_TOG, RGB_MOD, RGB_RMOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD,  KC_PSCR, KC_SCRL, KC_PAUS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_INS,  KC_HOME, KC_TRNS, 
+		         KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_DEL,  KC_END,  KC_TRNS, KC_TRNS, 
+		         KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS,                                      KC_TRNS, KC_TRNS, KC_TRNS
+	),
+
+	[2] = LAYOUT(
+		KC_ESC,  KC_GRV,  KC_1, 	KC_2, 		KC_3,   KC_4,   KC_5, KC_6, KC_7, KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, 
+		KC_PGUP, RGB_TOG, RGB_MOD, 	RGB_RMOD, 	RGB_HUI,   RGB_HUD,   RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD,    KC_PSCR,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, 
+		KC_PGDN, KC_CAPS, KC_A, 	KC_S, 		KC_D,   KC_F,   KC_G, KC_H, KC_J, KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT, 
+		         KC_LSFT, KC_Z, 	KC_X, 		KC_C,   KC_V,   KC_B, KC_B, KC_N, KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_UP,   KC_RSFT, 
+		         KC_LCTL, KC_LGUI,    	  		KC_LALT,  KC_SPC,  KC_SPC,    MO(1),                        KC_LEFT, KC_DOWN, KC_RGHT
```

Comment:
`MO(1)` won't do anything useful on layer 2; and the usefulness of this content of layer 2 in general is questionable anyway.